packages/itwin/measure-tools/src/tools/MeasureLocationTool.ts

```diff
@@ -204,8 +205,9 @@ MeasureLocationToolModel
       snapAperture: hit.viewport.pixelsFromInches(0.1),
     };
 
-    const result = await this.iModel.requestSnap(requestProps);
-    if (result.status !== SnapStatus.Success) return undefined;
+    const accuSnapInst = new AccuSnap();
+    const result = await accuSnapInst.doSnapRequest(hit);
```

Comment:
Wouldn't this be satisfied by using `IModelApp.accuSnap`, instead of creating a new instance of `AccuSnap`? Also looks like you should remove the requestProps variable.